### ISSUE TITLE
New update: Hal Daumé III - new citations

### ISSUE THREAD

**Issue by postmillenial** [postmillenial/reading-list#955]
From: scholaralerts-noreply@google.com<br>
Subject: Hal Daum&eacute; III - new citations<br>
First Attachment: http://ift.tt/19fNRe4<br>
Date: October 24, 2017 at 03:28AM<br><br><div style="font-family: arial,sans-serif; font-size: medium;">
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2i1F9LZ"><font size="3" color="#1A0DAB">Generative linguistics and neural networks at 60: foundation, friction, and fusion</font></a>
</h3>
<div><font size="2" color="#006621">J Pater - 2017</font></div>
<div><font size="2" color="#222222">Abstract. The birthdate of both generative linguistics and neural networks can be taken as<br>
1957, the year of the publication of seminal work by both Noam Chomsky and Frank<br>
Rosenblatt. This paper traces the development of these two approaches to cognitive<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gzdkdt" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2hZIOtm" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gyIzFm" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2i0FSwV"><font size="3" color="#1A0DAB">Presentation of the paper &ldquo;An architectural proposal to explore the data of a private community through visual analytic&rdquo;</font></a>
</h3>
<div><font size="2" color="#006621">J Dur&aacute;n-Escudero - 2017</font></div>
<div><font size="2" color="#222222">This is the presentation of the paper entitled &ldquo;An architectural proposal to explore the data of<br>
a private community through visual analytic&rdquo; in the Learning Analytics Track of the TEEM<br>
2017 International Conference held in C&aacute;diz (Spain) in October 18-20, 2017. In this<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gywsIu" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i0Tgku" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gz5fFC" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2yKye3j"><font size="3" color="#1A0DAB">Basic tasks of sentiment analysis</font></a>
</h3>
<div><font size="2" color="#006621">I Chaturvedi, S Poria, E Cambria&nbsp;- arXiv preprint arXiv:1710.06536, 2017</font></div>
<div><font size="2" color="#222222">Abstract: Subjectivity detection is the task of identifying objective and subjective sentences.<br>
Objective sentences are those which do not exhibit any sentiment. So, it is desired for a<br>
sentiment analysis engine to find and separate the objective sentences for further analysis,<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2y26PpO" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2yKyeAl" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2y1YhiV" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2i0bt1v"><font size="3" color="#1A0DAB">A semi-supervised domain adaptation assembling approach for image classification</font></a></h3>
<div><font size="2" color="#006621">S Saxena, S Pandey, P Khanna&nbsp;- Pattern Analysis and Applications</font></div>
<div><font size="2" color="#222222">Abstract Automatic annotation of images is one of the fundamental problems in computer<br>
vision applications. With the increasing amount of freely available images, it is quite possible<br>
that the training data used to learn a classifier has different distribution from the data which is<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gAqers" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i189mQ" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gzpzH7" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2i05BWc"><font size="3" color="#1A0DAB">Constructing Datasets for Multi-hop Reading Comprehension Across Documents</font></a>
</h3>
<div><font size="2" color="#006621">J Welbl, P Stenetorp, S Riedel&nbsp;- arXiv preprint arXiv:1710.06481, 2017</font></div>
<div><font size="2" color="#222222">Abstract: Most Reading Comprehension methods limit themselves to queries which can be<br>
answered using a single sentence, paragraph, or document. Enabling models to combine<br>
disjoint pieces of textual evidence would extend the scope of machine comprehension<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gxXWOx" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i0A32o" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gxieHP" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2i0cGpw"><font size="3" color="#1A0DAB">Improved Scoring Models for Semantic Image Retrieval Using Scene Graphs</font></a>
</h3>
<div><font size="2" color="#006621">ET Conser - 2017</font></div>
<div><font size="2" color="#222222">Abstract Image retrieval via a structured query is explored in Johnson, et al.[7]. The query is<br>
structured as a scene graph and a graphical model is generated from the scene graph's<br>
object, attribute, and relationship structure. Inference is performed on the graphical model<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gyLOwG" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i17Bxg" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gAatRF" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2i0VwZ7"><font size="3" color="#1A0DAB">Multi-view clustering with graph embedding for connectome analysis</font></a>
</h3>
<div><font size="2" color="#006621">G Ma, L He, CT Lu, W Shao, PS Yu, AD Leow&hellip;&nbsp;- Proceedings of the 26th&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">ABSTRACT Multi-view clustering has become a widely studied problem in the area of<br>
unsupervised learning. It aims to integrate multiple views by taking advantages of the<br>
consensus and complimentary information from multiple views. Most of the existing works in<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gxXVdr" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i1D9TF" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gywujA" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2hZsAAL"><font size="3" color="#1A0DAB">DeepGait: A Learning Deep Convolutional Representation for Gait Recognition</font></a></h3>
<div><font size="2" color="#006621">X Zhang, S Sun, C Li, X Zhao, Y Hu&nbsp;- Chinese Conference on Biometric Recognition, 2017</font></div>
<div><font size="2" color="#222222">Abstract Human gait, as a soft biometric, helps to recognize people by walking. To further<br>
improve the recognition performances, we propose a novel video sensor-based gait<br>
representation, DeepGait, using deep convolutional features. DeepGait is generated by<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gxJbLz" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i0E1Ij" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gxJciB" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2i0UUmm"><font size="3" color="#1A0DAB">A Semantically-Based Computational Approach to Narrative Structure</font></a>
</h3>
<div><font size="2" color="#006621">R Delmonte, G Marchesini&nbsp;- IWCS 2017&mdash;12th International Conference on&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract In this paper we will define narrative structure as characterized by a basic element,<br>
the narreme, which is here described as the basic unit of narrative structure, the smallest<br>
possible unit of the story. We annotated a full novel&ndash;" The Solid Mandala"(1966) by Nobel<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gxJcPD" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i0qsIR" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2gyT2k9" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><hr noshade size="1">
<font size="2" color="#222222">This Google Scholar Alert is brought to you by Google.<br><br><a href="http://ift.tt/2vUKjPi"><font color="#1A0DAB">Cancel alert</font></a><br><a href="http://ift.tt/2eWhNpX"><font color="#1A0DAB">List my alerts</font></a></font>
</div>
<br><br>
via Gmail
